### DISCUSSION TITLE
Suggestions for video content for the OKD community

### DISCUSSION THREAD

**Discussion by binnes** [okd-project/okd#1334] [Ideas]
If you have any suggestions for new video content for OKD please add to this discussion thread.

We want to create a backlog of video topics and so we can get a regular release cycle of new video content on OKD topic running.  This could be an idea you want to see or an offer to make a video.

**Comment by binnes** [comment]
This discussion is linked to an [OKD planning board item](https://github.com/okd-project/okd.io/issues/2)

Currently schedued topics:

(1) Debugging OKD Build Process/Demystifying Prow - Micheal McCune & Denis Moiseev - Recorded Sept 2
(2) Overview of OKD on Centos Stream CoreOS - Zack Zlotnik & Neal Gompa - Recorded Sept 9
(3) OKD SCOS Build Pipelines Explained - Luigi Mario Zuccarelli & Christian Glombek - Recorded Sept 16
(4) Introducing OKD Streams - Michelle Krejci & Christian Glombek - Recorded Sept 20
(5) OKD Operator Pipeline Explained - Luigi Mario Zuccarelli - Proposed for Sept 23
(6) Community Welcome and Overview Video - Jaime Magiera
(7) UPI OKD 4.x install (Single Node Master/Worker) - Luigi Mario Zuccarelli


Playlist so far: https://www.youtube.com/playlist?list=PLaR6Rq6Z4Iqck7Z0ekuJdsMU1fE6hkd6d

**Comment by dmueller2001** [comment]
Suggestion: Debugging a Cluster Bootstrap Failure

**Comment by elmiko** [comment]
i'd like to propose a video on [must-gather](https://github.com/openshift/must-gather) artifacts and why they are important.

storyboard
1. what are must-gathers
2. how do you make them
3. how do you interpret them
4. tools for making this easier
